invokeai/frontend/.prettierrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   trailingComma: 'es5',
   tabWidth: 2,
+  endOfLine: 'crlf',
```

Comment:
Setting this to `crlf` results in prettier changing a bazillion line endings. Any reason to not use `auto`? Our eslint rule already uses `auto` so I think it should match.